ui/src/reportAnIssue.ts

```diff
@@ -14,7 +15,14 @@ import { openUrl } from './utils/openUrl';
 export async function reportAnIssue(actionId: string, parsedError: IParsedError): Promise<void> {
     const { extensionName, extensionVersion, bugsUrl } = getPackageInfo();
 
-    let body: string = `
+    // Some browsers don't have very long URLs
```

Comment:
Not sure if you randomly picked 2000, but seems like the actual limit in several browsers. Suggest adding a link like this in the comment:

https://blogs.msdn.microsoft.com/ieinternals/2014/08/13/url-length-limits/

or

https://support.microsoft.com/help/208427/maximum-url-length-is-2-083-characters-in-internet-explorer

Reply:
Maybe from this?

https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Reply:
Nathan's was the one, but it all seemed to vague I hesitated to include a link.  will do so.